package.json

```diff
@@ -224,6 +224,11 @@
             "type": "array",
             "default": ["**/.*", "**/node_modules"],
             "description": "Specifies filepath pattern of folders to exclude while searching for maven projects."
+          },
+          "maven.executable.path": {
```

Comment:
Just curiosity, what is a setting's default ```scope``` value if the ```scope``` is not configured in package.json?